plex/server.py

```diff
@@ -35,8 +35,9 @@ def __init__(self, provider: xbmcmediaimport.MediaProvider):
         if not settings:
             raise ValueError('Invalid provider without settings')
 
-        self._localOnly = \
+        self._localOnly = bool(
```

Comment:
Is this really necessary?

Reply:
I can put it back, just used to '\' being considered bad form and preferring to do pretty much any other multi-solution other than '\' and changed it out of habit. Lmk if you want that changed back.

Reply:
Ah ok. It just looks redundant to me but if it's better code style I can live with it.